github/libp2p.yml

```diff
@@ -2495,6 +2495,50 @@ repositories:
     archived: true
   hydra-booster:
     archived: true
+  if-watch:
+    advanced_security: false
+    allow_update_branch: false
+    archived: false
+    branch_protection:
+      master:
+        allows_deletions: false
+        allows_force_pushes: false
+        blocks_creations: false
+        enforce_admins: false
+        lock_branch: false
+        require_conversation_resolution: false
+        require_signed_commits: false
+        required_linear_history: false
+        required_pull_request_reviews:
+          dismiss_stale_reviews: false
+          require_code_owner_reviews: false
+          required_approving_review_count: 1
+          restrict_dismissals: false
+        required_status_checks:
+          contexts:
+            - build
+          strict: false
+    default_branch: master
+    delete_branch_on_merge: false
+    description: Cross platform asynchronous network watcher
+    files:
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
+    has_discussions: false
+    merge_commit_message: PR_TITLE
+    merge_commit_title: MERGE_MESSAGE
+    secret_scanning_push_protection: true
+    secret_scanning: true
+    squash_merge_commit_message: COMMIT_MESSAGES
+    squash_merge_commit_title: COMMIT_OR_PR_TITLE
+    teams:
+      admin:
+        - rust-libp2p-maintainers
```

Comment:
Depending on https://github.com/libp2p/github-mgmt/pull/258, may have to be changed to `rust-libp2p Maintainers`.